platform-tests/src/test/java/org/eclipse/deeplearning4j/nd4j/linalg/workspace/DebugModeTests.java

```diff
@@ -134,6 +133,7 @@ public void testSpillMode_2(Nd4jBackend backend) {
             assertEquals(0, ws.getDeviceOffset());
 
             // array buffer should be spilled now
+            int shapeBufferAdd = backend.getClass().getName().toLowerCase().contains("cu") ? 0 : WorkspaceUtils.getShapeBufferRequireMemoryForWorkspace(array);
```

Comment:
Same note about the hack applies here too. 

If you need it in multiple places, put it into a helper method.

Reply:
Same answer as above.